x-pack/plugins/cases/server/client/cases/get.ts

```diff
@@ -114,15 +159,37 @@ export async function getTags({
 /**
  * Retrieves the reporters from all the cases.
  */
-export async function getReporters({
-  savedObjectsClient: soClient,
-  caseService,
-  logger,
-}: CasesClientArgs): Promise<User[]> {
+export async function getReporters(
+  params: AllReportersFindRequest,
+  clientArgs: CasesClientArgs
+): Promise<User[]> {
+  const {
+    savedObjectsClient: soClient,
+    caseService,
+    logger,
+    authorization: auth,
+    auditLogger,
+  } = clientArgs;
+
   try {
+    const queryParams = pipe(
+      excess(AllReportersFindRequestRt).decode(params),
+      fold(throwErrors(Boom.badRequest), identity)
+    );
+
+    const { filter: authorizationFilter } = await getAuthorizationFilter({
+      authorization: auth,
+      operation: Operations.getReporters,
+      auditLogger,
+    });
+
+    const filter = combineAuthorizedAndOwnerFilter(queryParams.owner, authorizationFilter);
```

Comment:
I think we need to call `ensureSavedObjectsAreAuthorized` for each of the cases that are returned right?

And then also the `logSuccessfulAuthorization` after as well.